.travis.yml

```diff
@@ -1,40 +1,32 @@
 language: rust
 sudo: false
 rust: nightly
+services: docker
 
 matrix:
   fast_finish: true
   include:
     - env: TARGET=i586-unknown-linux-gnu
     - env: TARGET=i686-unknown-linux-gnu
-    - env: TARGET=x86_64-unknown-linux-gnu NO_ADD=1
-    - env: TARGET=x86_64-unknown-linux-gnu-emulated NO_ADD=1 STDSIMD_TEST_EVERYTHING=1 FEATURES="intel_sde"
-    - env: TARGET=arm-unknown-linux-gnueabihf
+    - env: TARGET=x86_64-unknown-linux-gnu
     - env: TARGET=armv7-unknown-linux-gnueabihf
```

Comment:
I have removed arm-unknown-linux-gnueabihf temporarily, it is currently not supported by `cross` but a PR is in progress.